packages/lib/src/core/Services/payment-status.ts

```diff
@@ -12,10 +12,9 @@ export default function checkPaymentStatus(paymentData, clientKey, loadingContex
         throw new Error('Could not check the payment status');
     }
 
-    const options = {
+    return httpPost({
         loadingContext,
-        path: `services/PaymentInitiation/v1/status?clientKey=${clientKey}`
-    };
-
-    return httpPost(options, { paymentData });
+        path: `services/PaymentInitiation/v1/status?clientKey=${clientKey}`,
+        errorLevel: 'fatal'
```

Comment:
Adding the `errorLevel: 'fatal'` here would avoid polluting the console with the console.warn - but I am not fully sure that this is the best approach. 

It would end up throwing an `AdyenCheckoutError('NETWORK_ERROR', message')`, but this object won't be passed to the `onError` - it would be ignored [here](https://github.com/Adyen/adyen-web/blob/master/packages/lib/src/components/internal/Await/Await.tsx#L84) . 

Maybe a better approach is to use `errorLevel: 'silent'` , and let the Component retry the request (this way, there is no logging). If the timer of the Await runs out, then the `onError` will be triggered and your code can react upon that. What do you think of this approach?

Btw, do you know what is the reason for the request to the 'status' endpoint to fail?  Maybe we can try to find the root cause and fix that.

Reply:
>Btw, do you know what is the reason for the request to the 'status' endpoint to fail? Maybe we can try to find the root cause and fix that.

Reply:
Maybe we need to change status type from `network-error` to `error`, then `onError` will happen.
Also, checkPaymentStatus can raise Error for that code:
```
    if (!paymentData || !clientKey) {
        throw new Error('Could not check the payment status');
    }
```
So it in any case not a `network-error`. Wdyt?

Reply:
I'd say that start triggering the `onError` when there is a network problem can be a breaking change in this case. 

Merchants implement different use-cases when this callback is triggered (Some merchants redirect shoppers, other merchants display pop-ups, etc). We can't simply start triggering it if there is a network error. And, if such an error happens in this specific scenario, the frontend can ignore it and retry the request few seconds later due to the polling mechanism.

It seems that the feasible approach here is to use `errorLevel: 'silent'` and let the onError be triggered when the Component times out.

Reply:
Can you rebase on master? The issue with the i18n is already [fixed and released](https://github.com/Adyen/adyen-web/blob/master/packages/lib/src/components/internal/Await/Await.tsx#L83)

Reply:
Then it fixed? WE start getting another one error messages :)

Reply:
Now we get:

Reply:
Now we get:
```
Cannot read properties of undefined (reading 'get')
```

Reply:
Rebased anyway

Reply:
We have no more messages from our side yet. We also migrated to latest `adyen-web`. This `silent` thingy is worth now?

Reply:
Hey @aefimov . Thanks for getting back with the feedback. I'm glad that there are no error messages anymore.

I'd say that we do not need the `silent` change anymore - there is no one complaining about it, so let's keep as it is :)

Reply:
Ok, closing PR, then! Thanks for help!